src/siddhant/SwitchBank.java

```diff
@@ -0,0 +1,137 @@
+package siddhant;
+
+class SwitchBank{
+	String name;
+	int iamount = 30000;
+	int hamount = 25000;
+	int samount = 28000;
+	int bamount = 29000;
+	int idebitcount , icreditcount;
+	int hdebitcount , hcreditcount;
+	int sdebitcount , screditcount;
+	int bdebitcount , bcreditcount;
+	static int totalcreditcount = 0;
+	static int totaldebitcount = 0;
+	static int totalcount = 0;
+			
+	void operation(String Bank, String op , int amt){
+		switch(op){
+			case "Debit" :
+				switch(Bank){
+					case "icic" :
+						if(amt<=iamount){
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.